main.tf

```diff
@@ -37,6 +39,9 @@ locals {
   distinct_policy_names = distinct(compact(flatten([
     for k, v in local.spacelift_stacks : try(v.settings.spacelift.policies_by_name_enabled, var.policies_by_name_enabled) if v.enabled
   ])))
+
+  # Concatenate labels with infracost if it's enabled
+  labels = var.infracost_enabled ? concat(var.labels, ["infracost"]) : var.labels
```

Comment:
`var.labels` is not a variable for `main.tf`, it's in https://github.com/cloudposse/terraform-spacelift-cloud-infrastructure-automation/blob/master/modules/stack/variables.tf#L104

Reply:
It is a variable in `main.tf` in this PR, I added it to `variables.tf` ?

Do you mean it shouldn't be an input for this module ?

Reply:
oh sorry, did not notice the new variable